packages/core/src/browser/shell/application-shell.ts

```diff
@@ -1735,6 +1765,8 @@ export class ApplicationShell extends Widget {
         const area = this.currentWidget && this.getAreaPanelFor(this.currentWidget);
         if (area instanceof TheiaDockPanel && (area === this.mainPanel || area === this.bottomPanel)) {
             area.toggleMaximized();
+        } else {
```

Comment:
Is the log necessary?
Ideally the command should only be available if the 'toggle maximized' is available and will work.